--- a/meson.build
+++ b/meson.build
@@ -42,7 +42,8 @@ if get_option('buildtype') == 'release'
   )
 endif

-if get_option('native_optimizations')
-  add_project_arguments('-march=native', '-mtune=native', language: ['c', 'cpp'])
-endif
+# Forzar compatibilidad total en cualquier x86_64
+# Ignorar completamente native_optimizations
+add_project_arguments('-march=x86-64', '-mtune=generic', language: ['c', 'cpp'])
